docs/usage/self-hosted-experimental.md

```diff
@@ -39,3 +39,10 @@ If set to any string, Renovate will use this as the `user-agent` it sends with H
 
 If set to any value, Renovate will use a "hard" `process.exit()` once all work is done, even if a sub-process is otherwise delaying Node.js from exiting.
 See <https://github.com/renovatebot/renovate/issues/8660> for background on why this was created.
+
+## RENOVATE_X_SERVER_VERSION
+
+If set, Renovate will use this string as GitLab server version instead of checking via the GitLab API.
+This can be useful when you use the GitLab `CI_JOB_TOKEN` to authenticate Renovate.
+
+Read [platform details](modules/platform/gitlab/index.md) to learn why we need the server version on GitLab.
```

Comment:
I think there's some problem with the link to the `platform details`, I get a 404 page not found when I click the link in the GitHub preview.

Maybe we need a `../` in the path?

Reply:
No. the link should work when going live